packages/SwingSet/src/kernel/vatManager/supervisor-subprocess-xsnap.js

```diff
@@ -251,8 +253,7 @@ function makeWorker(port) {
   });
 }
 
-// @ts-ignore xsnap provides issueCommand global
```

Comment:
again, why is this no longer necessary? I suppose I should check change comments...

Reply:
As above.